### ISSUE TITLE
Add ahmia.fi Blacklist

### ISSUE THREAD

**Issue by austinsonger** [TheresAFewConors/Sooty#80]
**What is the feature that you are requesting?**
Detect a DNS Query of Onion Website with ahmia.fi blacklist

**Additional information**
WEBSITE
 ![](https://i.imgur.com/Dm8V6UT.png)

CHild Abuse Onion URL HASHes
![](https://i.imgur.com/qXNMrR9.png)

**Is your feature request related to a problem? Please describe.**
AlienVault didn't detect this and it would be nice to use Sooty to automatically hash the onion URL and then check the blacklist to see if the Onion URL is known for child abuse material.

#### URLS
- https://ahmia.fi/blacklist/
- https://ahmia.fi/blacklist/banned/
- https://github.com/ahmia/ahmia-index

**Comment by TheresAFewConors** [comment]
I think this would be a great addition. Are you currently working on this or is this a request for it to be added? Either way is fine.

**Comment by austinsonger** [comment]
> I think this would be a great addition. Are you currently working on this or is this a request for it to be added? Either way is fine.

Yeah i could. What section in Sooty would this be best to be placed?

**Comment by TheresAFewConors** [comment]
I think it would work under the reputation checker (option 3), since it essentially is performing that action. For now though, if you want to put it under its own option in the main menu (option 8 is free) it can then be moved later with the URL Scan option. 

Thanks!
